Turtlelord26 Melee 2.1 Unarmed Combat Experience Levels

Turtlelord26 opened the pull request:
Solves Issue #412 

* Consolidated Unarmed weapons to one Unarmed Strike
* Reintroduced Unarmed Combat Feats, with prerequisites and effects.
* Various Typography implied by the above changes
* Slight scope creep into Feat document readability

No mechanical effect is final; I didn't spend very long picking numbers. Unarmed Feat effects in particular are subject to change when we start working on Melee Weapon skill bonuses (a la the soon-to-be-eliminated-as-discrete-entities proficiency feats)

Turtlelord26 committed:
Unarmed Strike

Consolidated unarmed "weapons" into one Unarmed Strike
Removed references to mechanical bonuses from having higher training in unarmed combat (will re-add in feats)

Turtlelord26 committed:
Update 03_Feats.txt

Deleted "Monk" feat to consolidate the effect in the weapon skill bonuses.
Added newlines after tertiary headings for ease of repository scanning. Slight scope creep, mea culpa.
Added new feats Unarmed Combat Training/Expertise/Mastery/Grandmastery. Untrained unarmed combat is assumed by default for all characters (and has no bonuses/penalties attached, it is simply the default). These feats will provide numerical bonuses to unarmed strikes and unlock benefits in the Unarmed Combat Skill weapon bonuses list. Numbers not yet included.

Turtlelord26 committed:
Berserker Skill Passive Updated to New Feat

Turtlelord26 committed:
Typography

Turtlelord26 committed:
Unarmed Combat Feat Effects

Turtlelord26 committed:
Typography

bleehu commented on CharacterCreation/03_Feats.txt:
```diff
@@ -507,7 +549,59 @@ result in extra hit points, or take the average of the result (1d10 = 5,
+* Assassin Class and Level 5, OR
+* Berserker Class and Level 5
+
+	Increase base and minimum damage of Unarmed Strike by 20.
```
Is this an additional 20 on top of the extra damage from UACT, or is it the same one?

Turtlelord26 replied:
Additional...I just noticed that the feats don't have themselves as prereqs. It seems like Expert should require Training.

bleehu reviewed (approved):
I like the simplicity of it! Much more usable!

rosebrallier reviewed (approved):
Looks good on my front. I think your superbranch must have branched from dev before my adept purge; some of the changes you make here are already in dev.

bleehu commented:
I noticed that too, lol!

Turtlelord26 commented:
I'm thinking periodic merges from dev to the superbranch may be in order.

bleehu commented:
It's a good idea.

Turtlelord26 committed:
Unarmed Combat Feat Prerequisites

Now forms an explicit progression.

Turtlelord26 committed:
Merge branch 'Turtlelord26-Melee_2.1-Hand_to_Hand_Expertise' into dev

Turtlelord26 committed:
Merge pull request #444 from bleehu/dev

Dev -> Hand-to-Hand Expertise 4/11

bleehu commented on CharacterCreation/03_Feats.txt:
```diff
@@ -507,7 +549,60 @@ result in extra hit points, or take the average of the result (1d10 = 5,
+
+Prerequisites:
+* Unarmed Combat Mastery
+* STR 7, DEX 7
```
What do these comas mean? Do they mean 'and' or 'or'?

bleehu replied:
Should we have 'and' requirements each on their own bullet point?

Turtlelord26 replied:
And, will spell out.
Individual bullets are assumed to be ANDed with each other, but I thought it was a sane space-saving mechanism to list all attribute requirements on the same line.

bleehu commented on CharacterCreation/03_Feats.txt:
```diff
@@ -507,7 +549,60 @@ result in extra hit points, or take the average of the result (1d10 = 5,
+	Increase base and minimum damage of Unarmed Strike by 20.
+	
+	Whenever you would make a disarm attempt with Unarmed Combat 
+Training, you may instead make a Trip, Shove, or Grapple attempt.
```
See... what? for definition of shove, trip, etc?

Turtlelord26 replied:
Welll....those don't actually exist yet. But, per #366 they will exist before the superbranch goes to dev. I can include a reference to the maneuvers section of melee combat in basic rules.

bleehu replied:
Oh. Thought I was just forgetting where I left 'em. My bad.

Turtlelord26 commented:
Well that made a huge mess. Now I know not to manually resolve merges from dev to branches being merged into dev.

Turtlelord26 committed:
Revert "Dev -> Hand-to-Hand Expertise 4/11"

Turtlelord26 committed:
Merge pull request #445 from bleehu/revert-444-dev

Revert "Dev -> Hand-to-Hand Expertise 4/11"

bleehu commented on CharacterCreation/03_Feats.txt:
```diff
@@ -507,7 +549,60 @@ result in extra hit points, or take the average of the result (1d10 = 5,
+* STR 6, DEX 6
+* Level 10, OR Assassin Class and Level 5, OR Berserker Class and level 5
+
+	Increase base and minimum damage of Unarmed Strike by 20.
```
I think we should either say 'additional 20 damage' or list the combined total of the damage from melee feats. It helps reduce that ambiguity.

Turtlelord26 replied:
Can do

bleehu commented on CharacterCreation/03_Feats.txt:
```diff
@@ -507,7 +549,60 @@ result in extra hit points, or take the average of the result (1d10 = 5,
+	
+	Additionally, Once per round after a successful melee strike, 
+if the target is not wielding a CQB weapon, you may use a reaction to 
+make a disarm attempt.
```
I love that you've exempted CQB weapons here. It creates real meaning behind the choice to choose a CQB weapon or not. I'm impressed by your ability to keep track of so many dynamics at the same time.

Turtlelord26 replied:
I didn't realize how big that is while I was doing it, but you're correct about how much significance that adds to the choice. With that in mind, what would you say to switching the bonuses from Expertise and Mastery, in order to delay the point where having a CQB weapon doesn't save you?

bleehu replied:
It still costs 2 feats to remove that safety, and you still need to close to melee range; I think you put it right were it belongs personally. Make the design choice that you think works best, and we'll test it.

Turtlelord26 replied:
I think it's easier to tell if something's too powerful than too weak (if it's close to correct), so we'll test it with the earlier CQB disarm.

Turtlelord26 committed:
Merge pull request #446 from bleehu/Turtlelord-Melee_2.1_Project-Superbranch

Melee 2.1 Superbranch -> Hand-to-Hand Expertise 4/11

bleehu commented on CharacterCreation/03_Feats.txt:
```diff
@@ -507,7 +549,60 @@ result in extra hit points, or take the average of the result (1d10 = 5,
+	Whenever you would make a disarm attempt with Unarmed Combat 
+Training, you may instead make a Trip, Shove, or Grapple attempt.
+
+== Unarmed Combat Grand Mastery ==
```
The presence of late-game content makes me happy enough to headbang!

Turtlelord26 committed:
Addressing Review

Clarified "additional 20" in expertise, mastery, and grand mastery.
Explicitly ANDed Str and Dex requirements
References to Maneuvers included for ease of reference.

bleehu commented on CharacterCreation/02_Classes.txt:
```diff
@@ -377,6 +377,64 @@ Skill Proficiencies:
 ======================================
 
 
+=====   ADEPT   =====
```
Sorry to do this to ya, but we gotta re-re-yank Adept...

bleehu reviewed (changes requested):
Looks good, with the exception of a commit contamination. Almost there!

Turtlelord26 committed:
Cleaning commit junk part 1

Turtlelord26 committed:
Cleaning commit junk part 2

Turtlelord26 committed:
Cleaning commit junk part 3

Turtlelord26 committed:
Cleaning commit junk part 4

rosebrallier commented on CharacterCreation/03_Feats.txt:
```diff
@@ -525,42 +624,48 @@ Prerequisites:
 * Must be a Gunslinger
 
-	Enables the use of shotguns by Gunslingers.
+	Enables the use of shotguns by the listed classes.
```
It feels janky to use "the listed classes" for literally just one class. Is it cleaning up commit junk, because I deliberately changed this in PR#440

rosebrallier commented on CharacterCreation/03_Feats.txt:
```diff
@@ -525,42 +624,48 @@ Prerequisites:
 *  Must be a Paladin
 
-	Enables the use of Automatic Rifles by the listed class.
+	Enables the use of Automatic Rifles by the listed classes.
```
Commit junk? B/c the grammar burns.

rosebrallier commented on CharacterCreation/06_01_Systemized_Skills.txt:
```diff
@@ -141,6 +141,7 @@ Example for (STRENGTH) Athletics check:
 
 ===== Class Skill Proficiencies =====
 
+Adept: Knowledge - Nanites, Weapons - General
```
It BURNSSSSS

rosebrallier reviewed (commented):
I am so confused by everything.
Edit: Never mind, I figured out what was happening.

1sourcecontrol commented:
Keep in mind, this PR invalidates rules in misc. thrown weapons (further down in the weapons doc):

>	If you have Hand-to-Hand trained, +10 base damage for all misc thrown 
 weapons. Instead, if you have Hand-to-Hand Expert, +20 base damage for all misc 
 thrown weapons. 
>
 > 	Also, If you have melee combat skill >=20 then all DEX or STR requirements 
 for Misc. Thrown Weapons are reduced by 1.

Turtlelord26 commented:
Thank you for reminding me of those.

Turtlelord26 committed:
Patch Grenades and Thrown

Remove skill references and references to discontinued hand-to-hand experience levels.

bleehu commented on CharacterCreation/03_Feats.txt:
```diff
@@ -507,7 +549,63 @@ result in extra hit points, or take the average of the result (1d10 = 5,
+
+Prerequisites:
+* STR 5 and DEX 5
+* Levle 5, OR Assassin Class, OR Berserker Class
```
Levle?

Turtlelord26 replied:
*Own Head Explodes*

bleehu reviewed (changes requested):
One small typo, asking for a change before we make RoseB's Head explode, then shipit without needing another review. Great work!

Turtlelord26 committed:
Typography
